docs_user/modules/proc_deploying-the-block-storage-services.adoc

```diff
@@ -124,78 +121,66 @@ openstack volume service list
 ----
 
-. In this case you need to remove services for hosts `standalone.localdomain`
+. Remove services for hosts that are in the `down` state. For example:
 +
 ----
-oc exec -it cinder-scheduler-0 -- cinder-manage service remove cinder-backup standalone.localdomain
-oc exec -it cinder-scheduler-0 -- cinder-manage service remove cinder-scheduler standalone.localdomain
+$ oc exec -it cinder-scheduler-0 -- cinder-manage service remove cinder-backup standalone.localdomain
+$ oc exec -it cinder-scheduler-0 -- cinder-manage service remove cinder-scheduler standalone.localdomain
 ----
+
+. Apply the DB data migrations:
 +
-The reason why we haven't preserved the name of the backup service is because
-we have taken the opportunity to change its configuration to support
-Active-Active, even though we are not doing so right now because we have 1
-replica.
-//kgilliga: The above paragraph is confusing. Who changed the configuration? Unclear on what the replica refers to.
-
-. Now that the Block Storage services are running, the DB schema migration has been completed and you can proceed to apply the DB data migrations.
-While it is not necessary to run these data migrations at this precise moment,
-because you can run them right before the next upgrade, for adoption it is best to run them now to make sure there are no issues before running production workloads on the deployment.
-+
-The command to run the DB data migrations is:
+[NOTE]
+====
+You are not required to run the data migrations at this step, but you must run them before the next upgrade. However, for adoption, it is recommended to run the migrations now to ensure that there are no issues before you run production workloads on the deployment.
+====
 +
 ----
-oc exec -it cinder-scheduler-0 -- cinder-manage db online_data_migrations
+$ oc exec -it cinder-scheduler-0 -- cinder-manage db online_data_migrations
 ----
 
 .Verification
 
-Before you can run any checks you need to set the right cloud configuration for
-the `openstack` command to be able to connect to your {OpenShiftShort} control plane.
-
 . Ensure that the `openstack` alias is defined:
 +
 ----
-alias openstack="oc exec -t openstackclient -- openstack"
+$ alias openstack="oc exec -t openstackclient -- openstack"
 ----
 
-Now you can run a set of tests to confirm that the deployment is using your
-old database contents:
-
-. See that {block_storage} endpoints are defined and pointing to the control plane FQDNs:
+. Confirm that {block_storage} endpoints are defined and pointing to the control plane FQDNs:
 +
 ----
-openstack endpoint list --service cinderv3
+$ openstack endpoint list --service cinderv3
 ----
 
-. Check that the Block Storage services are running and up. The API won't show but if
-you get a response you know it's up as well:
+. Confirm that the Block Storage services are running. The API does not show but if you get a response you know it's up as well:
```

Comment:
Does this mean that the API does not show in the list of services after running the command?  And how do you get a response from the API in order to confirm that it's running?